apps/condo/domains/organization/schema/plugins/addOrganizationFieldPlugin.js

```diff
@@ -29,6 +29,11 @@ const addOrganizationFieldPlugin = ({ fromField, isRequired }) => plugin(({ fiel
                     resolvedData['organization'] = get(objWithOrganization, 'organization')
                 }
 
+                // If parent object is set to null -> set organization to null if it is possible
+                if (objWithOrganizationId === null && !isRequired) {
+                    resolvedData['organization'] = null
```

Comment:
Why should we explicitly specify that `organization = null`?
Why not `undefined`?

Reply:
It's a different case: you can specify null if you, for example, want to set the Resident not related to any organization